scripts/analyze.py

```diff
@@ -35,12 +35,11 @@ def parseFile(fileName):
 
 if __name__ == '__main__':
     args = sys.argv
-    if(len(args) != 4):
+    if(len(args) < 4):

```

Comment:
Have you already checkin the call change in shell file??
Does this 4 mean results in last 4 days? if so, should we make it configurable??

Reply:
Actually, 4 means at least py source file, day 1 res path, day 2 res path, output file name should be configed. And for any days larger than 2, it's ok. Not limited to 4.